src/features/Dashboard/Dashboard.actions.ts

```diff
@@ -0,0 +1,44 @@
+import * as api from "./Dashboard.api";
+import * as constants from "./Dashboard.constants";
+import { AppThunkDispatch } from "store/types";
+import { INDEX_OF_FIRST_ELEMENT } from "constants/common";
+import { LanguageType } from "store/language/types";
+
+export const getNowPlayingMovies = (activeLanguage: LanguageType) => async (dispatch: AppThunkDispatch) => {
+  try {
+    dispatch({ type: constants.GET_NOW_PLAYING_MOVIES_START });
+
+    const nowPlayingMovies = await api.getNowPlayingMovies(activeLanguage);
```

Comment:
Is it possible to provide limit to api, to fetch only `5`?

Reply:
No. 
![Screenshot from 2021-08-08 01-31-11](https://user-images.githubusercontent.com/46905606/128615378-c9da4002-60db-4e43-85b0-03891f999450.png)